src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,39 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers/add")
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/addDriverToCar.jsp")
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/drivers/add.jsp")
```
<img width="509" alt="image" src="https://user-images.githubusercontent.com/96740815/203334258-48fb9a32-a53a-4181-83c4-6e5d88439b33.png">